clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -1528,4 +1530,29 @@ public void testHandshakeUnwrapContinuesUnwrappingOnNeedUnwrapAfterAllBytesRead(
         assertEquals(SSLEngineResult.Status.OK, result.getStatus());
         assertEquals(SSLEngineResult.HandshakeStatus.NEED_WRAP, result.getHandshakeStatus());
     }
+
+    @Test
+    public void testSSLEngineCloseInboundInvokedOnClose() throws IOException {
+        // Given
+        SSLEngine sslEngine = mock(SSLEngine.class);
```

Comment:
Should we mock `closeInbound` to throw `SSLException` to test that we do catch the exception

Reply:
Thanks for the feedback! Addressed in 3eb2e99d91d51b313b5b224c9aa7dced1097fd30